integration-cli/docker_cli_images_test.go

```diff
@@ -18,6 +18,44 @@ func (s *DockerSuite) TestImagesEnsureImageIsListed(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestImagesEnsureImageWithTagIsListed(c *check.C) {
+	testRequires(c, Network)
```

Comment:
@cpuguy83 you're completely right, I don't remember why I put this and also think it's not needed… I will remove it :wink:.